torch/distributed/_functional_collectives.py

```diff
@@ -464,6 +483,13 @@ def all_reduce_coalesced(self: List[torch.Tensor], reduceOp: str, group: RANK_TY
     :: N.B. If you pass a PG or a 1D list to perform a MPMD collective, the compiler won't be able to recover
     that information and perform collective algebraic optimization. Use other forms of input for that.
     """
+    distributed_log.debug(
+        "all_reduce_coalesced sizes: [%s], reduce_op: %s, group: %s, tag: %s",
+        ", ".join([str(t.size()) for t in self]),
```

Comment:
don't do string formatting in the arg list. Use LazyString if you must.

Reply:
hmm, is it still too bad to prepare a list of sizes without string formatting?

Reply:
It's better. I personally draw the line at string formatting.